content/content/Usage/plugin management.md

```diff
@@ -43,10 +43,26 @@ Add any plugin to a plugin directory to be consumed by transform command.
 #### Example
 
 ```
-crane plugin-manager add foo --version 0.0.1 -p plugin-dir
+crane plugin-manager add foo --version 0.0.1
 ```
 
-The above command downloads the binary of plugin `foo` versioned `0.0.1` from appropriate source and places it in plugin directory `plugin-dir/managed` (default is `plugins`).
+The above command downloads the binary of plugin `foo` versioned `0.0.1` from appropriate source and by default places it in plugin directory `~/.local/share/crane/plugins`
+
+A directory to install the plugin can optionally be specified:
+```
+sudo crane plugin-manager add OpenShiftPlugin -p /usr/local/share/crane/plugins
+```
+
+### Plugin loading behavior
+Crane looks for plugins in four directories. The order of precedence is:
+1. `./plugins`
+1. `~/.local/share/crane/plugins` - This can be overriden with the -p option when running a transform.
+1. `/usr/local/share/crane/plugins`
+1. `/usr/share/crane/plugins`
```

Comment:
ordering is a little messed up here. The numbers need to be 1. 2. 3. 4.

Reply:
Markdown handles this:
![image](https://user-images.githubusercontent.com/1850500/182168631-5edea791-fed9-466e-8540-4fbc64909b6b.png)